File: src/GestionNiveauxBundle/Entity/classe.php
<?php

namespace GestionNiveauxBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * classe
 *
 * @ORM\Table(name="classe")
 * @ORM\Entity(repositoryClass="GestionNiveauxBundle\Repository\classeRepository")
 */
class classe
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;
    /**
     * @ORM\ManyToOne(targetEntity="niveau", inversedBy="Classes" )
     * @ORM\JoinColumn(name="niveau_id", referencedColumnName="id" )
     */
    private $niveau;

    /**
     * @return mixed
     */
    public function getNiveau()
    {
        return $this->niveau;
    }

    /**
     * @param mixed $niveau
     */
    public function setNiveau($niveau)
    {
        $this->niveau = $niveau;
    }

    /**
     * @var int
     *
     * @ORM\Column(name="num", type="integer")
     */
    private $num;

    /**
     * @return int
     */
    public function getNom()
    {
        return $this->nom;
    }

    /**
     * @param int $nom
     */
    public function setNom($nom)
    {
        $this->nom = $nom;
    }

    /**
     * @var int
     *
     * @ORM\Column(name="nom", type="string")
     */

    private $nom;


    /**
     * @var int
     *
     * @ORM\Column(name="nbrEtudiant", type="integer")
     */
    private $nbrEtudiant;


    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set num
     *
     * @param integer $num
     *
     * @return \GestionNiveauxBundle\Entity\classe
     */
    public function setNum($num)
    {
        $this->num = $num;

        return $this;
    }

    /**
     * Get num
     *
     * @return int
     */
    public function getNum()
    {
        return $this->num;
    }

    /**
     * Set nbrEtudiant
     *
     * @param integer $nbrEtudiant
     *
     * @return classe
     */
    public function setNbrEtudiant($nbrEtudiant)
    {
        $this->nbrEtudiant = $nbrEtudiant;

        return $this;
    }

    /**
     * Get nbrEtudiant
     *
     * @return int
     */
    public function getNbrEtudiant()
    {
        return $this->nbrEtudiant;
    }
    public  function __toString()
    {
        return(string)$this->getNom();
    }

}

File: EdPlus_Desktop_Eya/src/entities/Classe.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

/**
 *
 * @author Eya
 */
public class Classe {

    private int id;
    private int niveau_id;
    private int num;
    private int nbrEtudiant;
    private String nom;

    public Classe() {
    }

    public Classe(int id, int niveau_id, int num, int nbrEtudiant, String nom) {
        this.id = id;
        this.niveau_id = niveau_id;
        this.num = num;
        this.nbrEtudiant = nbrEtudiant;
        this.nom = nom;
    }

    public Classe(int num, int nbrEtudiant, String nom) {

        this.num = num;
        this.nbrEtudiant = nbrEtudiant;
        this.nom = nom;
    }

    public Classe(int niveau_id, int num, int nbrEtudiant, String nom) {
        this.niveau_id = niveau_id;
        this.num = num;
        this.nbrEtudiant = nbrEtudiant;
        this.nom = nom;
    }

    @Override
    public String toString() {
        return "Classe{" + "id=" + id + ", niveau_id=" + niveau_id + ", num=" + num + ", nbrEtudiant=" + nbrEtudiant + ", nom=" + nom + '}' + "\n";
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setNiveau_id(int niveau_id) {
        this.niveau_id = niveau_id;
    }

    public void setNum(int num) {
        this.num = num;
    }

    public void setNbrEtudiant(int nbrEtudiant) {
        this.nbrEtudiant = nbrEtudiant;
    }

    public int getId() {
        return id;
    }

    public int getNiveau_id() {
        return niveau_id;
    }

    public int getNum() {
        return num;
    }

    public int getNbrEtudiant() {
        return nbrEtudiant;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

}
